satpy/readers/ahi_hsd.py

```diff
@@ -33,22 +33,26 @@
 """
 
 import logging
+import os
+import warnings
 from datetime import datetime, timedelta
 
-import numpy as np
 import dask.array as da
+import numpy as np
 import xarray as xr
-import warnings
-import os
 
 from satpy import CHUNK_SIZE
-from satpy.readers.file_handlers import BaseFileHandler
-from satpy.readers.utils import unzip_file, get_geostationary_mask, \
-                                np2str, get_earth_radius, \
-                                get_user_calibration_factors, \
-                                apply_rad_correction
-from satpy.readers._geos_area import get_area_extent, get_area_definition
 from satpy._compat import cached_property
+from satpy.readers._geos_area import get_area_definition, get_area_extent
+from satpy.readers.file_handlers import BaseFileHandler
+from satpy.readers.utils import (
+    apply_rad_correction,
+    get_earth_radius,
+    get_geostationary_mask,
+    get_user_calibration_factors,
+    np2str,
+    unzip_file,
+)
```

Comment:
But where it _does_ tolerate having six imports from one module, spread over eight lines.  What's the difference with the cases where it insists on repeating the module the names are imported from?

Reply:
So this may be an inconsistency when isort (the tool reordering the imports) wasn't able to parse the config that @mraspaud had originally made. So in the modifiers file it split them across multiple lines which is against what we configured it for. Or vice versa. It is definitely something we should play with and see what it does. I agree with Martin in the other comment that sometimes I think it is a line length thing.